src/solvers/dgsem_tree/dg_1d_parabolic.jl

```diff
@@ -0,0 +1,502 @@
+
+    # Compute interface flux as mean of left and right viscous fluxes
+    # TODO: parabolic; only BR1 at the moment
+    flux = 0.5 * (flux_ll + flux_rr)
+
+    # Copy flux to left and right element storage
+    for v in eachvariable(equations_parabolic)
+        surface_flux_values[v, left_direction,  left_id]  = flux[v]
+        surface_flux_values[v, right_direction, right_id] = flux[v]
+    end
+  end
+
+  return nothing
+end
+
+
+# This is the version used when calculating the divergence of the viscous fluxes
+function prolong2boundaries!(cache_parabolic, flux_viscous,
+                             mesh::TreeMesh{1}, equations_parabolic::AbstractEquationsParabolic,
+                             surface_integral, dg::DG, cache)
+  @unpack boundaries = cache_parabolic
+  @unpack neighbor_sides = boundaries
+
+  @threaded for boundary in eachboundary(dg, cache_parabolic)
+    element = boundaries.neighbor_ids[boundary]
+
+    if neighbor_sides[boundary] == 1
+        # element in -x direction of boundary
+        for v in eachvariable(equations_parabolic)
+          # OBS! `boundaries.u` stores the interpolated *fluxes* and *not the solution*!
+          boundaries.u[1, v, boundary] = flux_viscous[v, nnodes(dg), element]
+        end
+      else # Element in +x direction of boundary
+        for v in eachvariable(equations_parabolic)
+          # OBS! `boundaries.u` stores the interpolated *fluxes* and *not the solution*!
+          boundaries.u[2, v, boundary] = flux_viscous[v, 1,          element]
+        end
+      end
+    end
+
+  return nothing
+end
+
+
+function calc_viscous_fluxes!(flux_viscous, gradients, u_transformed, mesh::TreeMesh{1},
+                              equations_parabolic::AbstractEquationsParabolic,
+                              dg::DG, cache, cache_parabolic)
+
+  @threaded for element in eachelement(dg, cache)
+    for i in eachnode(dg)
+      # Get solution and gradients
+      u_node = get_node_vars(u_transformed, equations_parabolic, dg, i, element)
+      gradients_1_node = get_node_vars(gradients, equations_parabolic, dg, i, element)
+
+      # Calculate viscous flux and store each component for later use
+      flux_viscous_node = flux(u_node, gradients_1_node, 1, equations_parabolic)
+      set_node_vars!(flux_viscous, flux_viscous_node, equations_parabolic, dg, i, element)
+    end
+  end
+end
+
+
+function calc_boundary_flux_gradients!(cache, t, boundary_conditions_parabolic::BoundaryConditionPeriodic,
+                                      mesh::TreeMesh{1}, equations_parabolic::AbstractEquationsParabolic,
+                                      surface_integral, dg::DG)
+  return nothing
+end
+
+function calc_boundary_flux_divergence!(cache, t, boundary_conditions_parabolic::BoundaryConditionPeriodic,
+                                        mesh::TreeMesh{1}, equations_parabolic::AbstractEquationsParabolic,
+                                        surface_integral, dg::DG)
+  return nothing
+end
```

Comment:
Since these are the same as the 2D/3D case, we could replace these with general versions in the future.

Reply:
@SigrunOrtleb Could you please open an issue for this? There is no need to change it right now but we should keep track of it to discuss it later.

Reply:
#1418